api/reports.py

```diff
@@ -5,59 +5,76 @@
 from pydantic import BaseModel
 from typing import List, Optional
 from libs.utils.reports import ReportsService
-from pathlib import Path
 
 reports_api = FastAPI()
 
 reports = ReportsService()
 
+
 class HourlyReport(BaseModel):
     hours: List[int]
     detected_objects: List[float]
     violating_objects: List[float]
+    detected_faces: List[float]
+    faces_with_mask: List[float]
+
     class Config:
         schema_extra = {
             'example': [{
                 'hours': list(range(0, 23)),
                 'detected_objects': [0., 7273.5, 10011.9, 0.5, 0., 7273., 10011., 0., 0., 7273., 10011., 0.,
                                      0., 7273.1, 10011.2, 0.5, 0., 7273., 10011., 0., 0., 7273., 10011., 0.],
                 'violating_objects': [0., 4920.5, 6701.3, 0.5, 0., 4920., 6701., 0., 0., 4920., 6701., 0.,
-                                      0., 4920.1, 6701.44, 0.1, 0., 4920., 6701., 0., 0., 4920., 6701., 0.]
+                                      0., 4920.1, 6701.44, 0.1, 0., 4920., 6701., 0., 0., 4920., 6701., 0.],
+                'detected_faces': [0., 7273.5, 10011.9, 0.5, 0., 7273., 10011., 0., 0., 7273., 10011., 0.,
```

Comment:
I know this is an example, but this hourly report returns the totals by hour right? Or does it return averages?

Reply:
It returns averages, however, we were discussing changing it to rounded integers (of the mean)

Reply:
I changed it to integer in de8330d59513512bb3366c98a53d41c494cdeb23